Source/Scene/Billboard.js

```diff
@@ -103,6 +103,7 @@ define([
         this._translucencyByDistance = options.translucencyByDistance;
         this._pixelOffsetScaleByDistance = options.pixelOffsetScaleByDistance;
         this._heightReference = defaultValue(options.heightReference, HeightReference.NONE);
+        this._sizeInMeters = defaultValue(options.sizeInMeters, false);
```

Comment:
It doesn't need to be part of this pull request (maybe for a code sprint), but billboards are starting to get "fat" enough that we should consider storing properties like this in a bitmask, and then converting them in the `set`/`get` functions.